packages/wix-ui-tpa/src/components/Image/Image.st.css

```
@@ -110,3 +118,32 @@
 .root.overrideStyleParams .image {
   opacity: value(imageOpacity);
 }
+
+.root.isError {
+  background-color: value(errorBackgroundColor);
+  position: relative;
+}
+
+.root .errorWrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  flex-direction: column;
+  background: rgba(0, 0, 0, 0.6);
+  height: 100%;
+  width: 100%;
+  z-index: 1;
+}
+
+.errorMessage {
+  -st-extends: TPAText;
+  -st-mixin: TPAText(MainTextColor value(errorTextColor), MainTextFont font(errorFontStyle));
```

The reason that the styles aren't wired correctly is because the usage. It should be:
```suggestion
  -st-mixin: TPAText(MainTextColor value(errorTextColor), MainTextFont value(errorFontStyle));
```

We needed to get the value from the variable. 😬

Also, `font` is already invoked inside the Text component:
https://github.com/wix/wix-style-react/blob/master/packages/wix-ui-tpa/src/components/Text/Text.st.css#L51